index.js

```diff
@@ -1,32 +1,19 @@
-var stream = require('readable-stream')
-var util = require('util')
-var generator = require('./generator')
+const { Transform } = require('streamx')
+const MerkleGenerator = require('./generator')
 
-module.exports = MerkleTree
+module.exports = class MerkleTreeStream extends Transform {
+  constructor (opts, roots) {
+    super({objectMode: true, highWaterMark: (opts && opts.highWaterMark) || 16})
```

Comment:
no object mode in streamx so we can remove that one

Reply:
Fixed in https://github.com/mafintosh/merkle-tree-stream/pull/7/commits/e97977ed3d53cf5cfd1afbb0f1a3de63ef0c2d00 (also replaced `new Buffer('...')` with `Buffer.from('...')`)